keyboards/ai03/lunar/keymaps/muzfuz/readme.md

```diff
@@ -0,0 +1,8 @@
+# muzfuz's keymap for Lunar
+
+```shell
+make ai03/lunar:muzfuz
```

Comment:
The `:dfu` target should work for this, actually: 

```suggestion
make ai03/lunar:muzfuz:dfu
```

Reply:
Great!